extension-manifest-v3/src/store/options.js

```diff
@@ -29,6 +29,7 @@ export const SYNC_OPTIONS = [
   'trackerCount',
   'wtmSerpReport',
   'serpTrackingPrevention',
+  'experimentalCosmeticFilters',
```

Comment:
I would name this option just `experimental` - as this not mean only cosmetics or filters (at all for DNR version)
You have even a bug here, as this key does not match the below one ;)

Reply:
Prefer to call it `experimentalFilters` as I expect we can have different experiments in near feature